src/commands/env.rs

```diff
@@ -62,13 +72,23 @@ impl Command for Env {
             .shell
             .or_else(&infer_shell)
             .ok_or(Error::CantInferShell)?;
-        let multishell_path = make_symlink(config);
+
+        let multishell_path = {
+            if config.multishell_path().is_none() {
+                make_symlink(config)
+            } else {
+                config.multishell_path().unwrap().to_path_buf()
+            }
+        };
```

Comment:
as I mentioned in a previous comment, I am not sure about it. Maybe this could be a flag (`--force-create-symlink` or `--ignore-current-env`, not sure about the "polarity" or the namings tho)

anyway, a more rusty way of doing that, would be:

```suggestion
        let multishell_path = match config.multishelll_path() {
            None => make_symlink(config),
            Some(p) => p.to_path_buf(),
        };
```

or

```suggestion
        let multishell_path = if let Some(p) = config.multishell_path() {
            p.to_path_buf()
        } else {
            make_symlink(config)
        };
```

or something with `.map` or `.unwrap_or_else` and all these `Option` functions 😁

Reply:
what do you think about introducing a flag? what polarity do you think it should have?
I lean towards `--ignore...` because it's an optimization: if we have `--force-...` then all tmux users (or any other terminal multiplexer) will share the node version across all shell sessions

Reply:
I think I'm too worried about the temp dir 😅 So I will revert the code and focus on the duplicate path variable.